src/main/java/duke/Task.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+public class Task {
+    protected String name;
+    protected boolean isDone;
+    protected String taskType;
+
+    public Task(String name, String taskType) {
+        this.name = name;
+        this.isDone = false;
+        this.taskType = taskType;
+    }
+
+    public Task(String name, String taskType, boolean isDone) {
+        this.name = name;
+        this.taskType = taskType;
+        this.isDone = isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X

```

Comment:
maybe the comment style can be improved.

Reply:
I have updated with a JavaDoc comment. Thank you for the feedback :)